.github/workflows/ci.yml

```diff
@@ -105,6 +105,14 @@ jobs:
           name: tests
           path: _build/default/pool/test/
 
+      - name: Notify about failure
```

Comment:
Could you extract this into an alias so we can use the alias here and on L146 (to reduce duplicated code)

(Assuming this and the bottom one are the same)

Reply:
Support yaml anchors is still open: https://github.com/actions/runner/issues/1182